### PR TITLE
Adds new Partner Highlight blog on semantic search. 

### PR DESCRIPTION
**Pull request by dtaivpp**
### Description
Adds new Partner Highlight blog on semantic search with several resources including code repos and videos. 

### Issues Resolved
N/A

### Check List
- [X] Commits are signed per the DCO using --signoff

By submitting this pull request, I confirm that my contribution is made under the terms of the BSD-3-Clause License.

### CODE REVIEW

**Comment by dtaivpp** [comment]
Pinging @pajuric and @natebower for review.

**Comment by dtaivpp** [comment]
@pajuric Fixed the namign

File: _posts/2024-05-30-do-you-need-semantic-search.markdown [hunk 1]
```diff
@@ -0,0 +1,24 @@
...
+  - dtaivpp
+date:   2024-05-30 11:45 -0700
+categories:
+  - technical-posts
+meta_keywords: opensearch hybrid search, opensearch semantic search, vector search
+meta_description: You might be asking yourself do you really need semantic search? I've been asked this question dozens of times so lets dive in and see what semantic search is and why you might want to use it. 
+SemanticSearch: lpQiJGpeeWU
+---
+
+I get ask this question all the time. Do I need semantic search? It's a good question but it's not the right question to start with. We need to first understand what semantic search does for us. Then we can dive into whether or not it's a good fit for your use case.
```
Comment (natebower) [inline _posts/2024-05-30-do-you-need-semantic-search.markdown:14]:
```suggestion
I get asked this question all the time: Do I need semantic search? It's a good question, but it's not the right question to start with. You need to first understand what semantic search can do for you. Then you can dive into whether or not it's a good fit for your use case.
```

File: _posts/2024-05-30-do-you-need-semantic-search.markdown [hunk 2]
```diff
@@ -0,0 +1,24 @@
...
+
+I've answered both questions in my [most recent blog](https://tippybits.com/should-you-be-doing-vector-search/) and its video counterpart linked below. These answer the following questions:
```
Comment (natebower) [inline _posts/2024-05-30-do-you-need-semantic-search.markdown:16]:
```suggestion
I've answered both questions in my [most recent blog post](https://tippybits.com/should-you-be-doing-vector-search/) and its video counterpart, linked below. These answer the following questions: 
```

File: _posts/2024-05-30-do-you-need-semantic-search.markdown [hunk 3]
```diff
@@ -0,0 +1,24 @@
...
+meta_description: You might be asking yourself do you really need semantic search? I've been asked this question dozens of times so lets dive in and see what semantic search is and why you might want to use it. 
+SemanticSearch: lpQiJGpeeWU
+---
+
+I get ask this question all the time. Do I need semantic search? It's a good question but it's not the right question to start with. We need to first understand what semantic search does for us. Then we can dive into whether or not it's a good fit for your use case.
+
+I've answered both questions in my [most recent blog](https://tippybits.com/should-you-be-doing-vector-search/) and its video counterpart linked below. These answer the following questions: 
+
+1. How does OpenSearch find documents out of the box?
+2. Why these traditional search methods don't always work.
```
Comment (natebower) [inline _posts/2024-05-30-do-you-need-semantic-search.markdown:19]:
```suggestion
2. Why don't these traditional search methods always work? 
```

File: _posts/2024-05-30-do-you-need-semantic-search.markdown [hunk 4]
```diff
@@ -0,0 +1,24 @@
...
+SemanticSearch: lpQiJGpeeWU
+---
+
+I get ask this question all the time. Do I need semantic search? It's a good question but it's not the right question to start with. We need to first understand what semantic search does for us. Then we can dive into whether or not it's a good fit for your use case.
+
+I've answered both questions in my [most recent blog](https://tippybits.com/should-you-be-doing-vector-search/) and its video counterpart linked below. These answer the following questions: 
+
+1. How does OpenSearch find documents out of the box?
+2. Why these traditional search methods don't always work. 
+3. When and how to use semantic search in OpenSearch.
```
Comment (natebower) [inline _posts/2024-05-30-do-you-need-semantic-search.markdown:20]:
```suggestion
3. When and how should semantic search be used in OpenSearch? 
```

File: _posts/2024-05-30-do-you-need-semantic-search.markdown [hunk 5]
```diff
@@ -0,0 +1,24 @@
...
+
+I've answered both questions in my [most recent blog](https://tippybits.com/should-you-be-doing-vector-search/) and its video counterpart linked below. These answer the following questions: 
+
+1. How does OpenSearch find documents out of the box?
+2. Why these traditional search methods don't always work. 
+3. When and how to use semantic search in OpenSearch. 
+
+To help I created some additional resources to help you get started! [This GitHub Gist](https://gist.github.com/dtaivpp/c587d99a2cab441eba0314534ae87c86) will help you create an OpenSearch docker compose environment. Also, I have created [this Gist](https://gist.github.com/dtaivpp/d7e8d8a3ee5debaf896ed2f45b915ad3) with all the code needed to create your own semantic search pipeline right from OpenSearch Dashboards. I hope you enjoy them and if your company needs help getting started with semantic search they can reach out to me [on my website](https://tippybits.com/services)!
```
Comment (natebower) [inline _posts/2024-05-30-do-you-need-semantic-search.markdown:22]:
```suggestion
I have created some additional resources to help you get started! [This GitHub Gist](https://gist.github.com/dtaivpp/c587d99a2cab441eba0314534ae87c86) will help you create an OpenSearch Docker Compose environment. I have also created [this Gist](https://gist.github.com/dtaivpp/d7e8d8a3ee5debaf896ed2f45b915ad3) with all the code needed to create your own semantic search pipeline right from OpenSearch Dashboards. I hope you enjoy them, and if your organization needs help getting started with semantic search, you can reach out to me [on my website](https://tippybits.com/services).
```

Comment (natebower) [inline _posts/2024-05-30-do-you-need-semantic-search.markdown:22]: Minor suggestion: "find them useful" instead of "enjoy them"?

**Review by natebower — COMMENTED** [review]
@dtaivpp Please see my comment (just one 😄) and changes and let me know if you have any questions. Thanks!

@pajuric This is ready to go once David indicates that he has addressed my changes and is happy with the draft.

**Comment by dtaivpp** [comment]
@krisfreedain, @pajuric, @nateynateynate, any updates on this?

**Comment by pajuric** [comment]
@dtaivpp - I plan to launch this tomorrow. Please update the date to 6/5/24.

**Comment by pajuric** [comment]
@nateynateynate @krisfreedain - This blog is cleared to launch. Please publish.

**Review by krisfreedain — APPROVED** [review]
LGTM!

**Pull request merged**